.idea/codeStyles/Project.xml

```diff
@@ -1,248 +1,30 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
-    <option name="IGNORE_SAME_INDENTS_FOR_LANGUAGES" value="true" />
-    <option name="OTHER_INDENT_OPTIONS">
-      <value>
-        <option name="INDENT_SIZE" value="2" />
-        <option name="CONTINUATION_INDENT_SIZE" value="4" />
-        <option name="TAB_SIZE" value="2" />
```

Comment:
Why have you deleted this? Revert these changes.

Reply:
I don't delete this because i didn't open it while fixing

Reply:
how do i revert kindly guide me?

Reply:
sir i have update my commit actually i committed the whole directory in which project.xml got modified now i committed adding files individually but travis build error is coming.Can you please help me to resolve it?